README.md

```diff
@@ -1,14 +1,13 @@
 # First run
 
-Clone kpi and kobocat repos in this directory
+Clone kpi repo in this directory
 
 1. Build `docker compose build --pull`
 1. Start postgres `docker compose up postgres` this ensures it has time to initialize
-1. Migrate kobocat `docker compose run --rm kobocat ./manage.py migrate`
-1. Migrate kpi `docker compose run --rm kpi ./manage.py migrate`
+1. Run Django database migrations `docker compose run -rm kpi scripts/migrate.sh`
 1. Make user `docker compose run --rm kpi ./manage.py createsuperuser`
-1. Edit `/etc/hosts` and add `127.0.0.1 kf.kobo.local kc.kobo.local ee.kobo.local`
-1. Run `npm i --force` in the kpi directory.
```

Comment:
Unrelated, but we no longer need it

Reply:
Cannot comment on [line 37](https://github.com/kobotoolbox/kobo-compose/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R37), but we should change:

> python packages in kpi or kobocat change, you can build like this

for 

> python packages in kpi change, you can build like this